packages/migration/src/index.ts

```diff
@@ -107,6 +111,24 @@ export class CommandMigrationFactory {
       return sourceFile
     }
 
+    private migrateHerokuCliUtilsExports(sourceFile: ts.SourceFile, file: string): ts.SourceFile {
+      const importName = findRequiredPackageVarNameIfExits(sourceFile, 'heroku-cli-util')
```

Comment:
If you set a breakpoint and look at the `SourceFile` object, there is a list of imports extracted from the statements and available as a separate property. This could allow you to shortcut the process of traversing the AST and simplify the process.

Reply:
I am doing something similar in `findRequiredPackageVarNameIfExits` by only looking at file-level declarations. Unfortunately, `SourceFile.imports` doesn't tell me the variable name, so I look at `Sourcefile.statements[i].declarationList.declarations[n].name.escapedText`